README.md

```diff
@@ -143,21 +144,50 @@ docker run -it --rm \
   -w $(realpath "${PWD}") \
   --network host \
   vagrantlibvirt/vagrant-libvirt:latest \
-    vagrant status
+  vagrant status
 ```
 
-It's possible to define an alias in `~/.bashrc`, for example:
+It's possible to define a function in `~/.bashrc`, for example:
 ```bash
-alias vagrant='
+vagrant(){
   docker run -it --rm \
-    -e LIBVIRT_DEFAULT_URI \
-    -v /var/run/libvirt/:/var/run/libvirt/ \
-    -v ~/.vagrant.d:/.vagrant.d \
-    -v $(realpath "${PWD}"):${PWD} \
-    -w $(realpath "${PWD}") \
-    --network host \
-    vagrantlibvirt/vagrant-libvirt:latest \
-    vagrant'
+  -e LIBVIRT_DEFAULT_URI \
+  -v /var/run/libvirt/:/var/run/libvirt/ \
+  -v ~/.vagrant.d:/.vagrant.d \
+  -v $(realpath "${PWD}"):${PWD} \
+  -w $(realpath "${PWD}") \
+  --network host \
+  vagrantlibvirt/vagrant-libvirt:latest \
+  vagrant $@
+}
+
+```
+
+## Using Podman based Installation
+To run with Podman you need to include
+
+```bash
+  --entrypoint /bin/bash \
+  --security-opt label=disable \
+```
+
+for example:
+
+```bash
+vagrant(){
+  podman run -it --rm \
+  -e LIBVIRT_DEFAULT_URI \
+  -v /var/run/libvirt/:/var/run/libvirt/ \
+  -v ~/.vagrant.d/boxes:/vagrant/boxes \
+  -v ~/.vagrant.d/data:/vagrant/data \
```

Comment:
I think probably a good idea to add an explanation below about this being different to the docker command, basically that since podman has the root user in the container mapped to the host user, it's possible to simply mount the needed data directories directly onto the target paths. While in the docker one it is mounting them to `/.vagrant.d` and the entrypoint is handling the mapping as part of setting up the correct user.

It's not perfect, because `~/.vagrant.d/Vagrantfile` won't be picked up in the current example if someone is using it, but perhaps such a note can help them know to add that to the list of the volume mounts.

Reply:
How about this explanation:

"Running Podman in rootless mode maps the `root` user inside the container to your host user so we need to bypass `entrypoint.sh` and mount persistent storage directly to `/vagrant`."

I did a bit of testing and was able to use a Vagrantfile in `~/.vagrant.d/` without problem.

Reply:
That sounds good. I wasn't expecting that it could pick up the user `~/.vagrant.d/Vagrantfile` because I thought it used `ENV['VAGRANT_HOME']/Vagrantfile`, where the env var `VAGRANT_HOME` is altered inside the container to be `/vagrant/`, so that's good to know that it works.

Reply:
It should pickup whatever Vagrantfile is the current directory which is the purpose of 

```
  -v $(realpath "${PWD}"):${PWD} \
  -w $(realpath "${PWD}") \
```

right?

Reply:
I think we're talking about 2 different Vagrantfile's. When vagrant loads it looks in multiple locations for Vagrantfiles and combines them together, so it reads and processes in the following order:
* `<box>/Vagrantfile`
* `${PWD}/Vagrantfile`
* `${VAGRANT_HOME}/Vagrantfile`

By default VAGRANT_HOME resolves to ~/.vagrant.d, which is also where the plugins get installed under.

As an example I have the following Vagrantfile in that location:
```
Vagrant.configure("2") do |config|
  config.vm.provider :libvirt do |libvirt|
    libvirt.storage_pool_name = "vms"
  end

  if Vagrant.has_plugin?("vagrant-proxyconf")
    if !ENV['VAGRANT_ENABLE_PROXY'].nil?
        config.proxy.enabled  = true
        config.proxy.http     = ENV['http_proxy'] || ENV['HTTP_PROXY'] || nil
        config.proxy.https    = config.proxy.http
        config.proxy.no_proxy = ENV['no_proxy'] || ENV['NO_PROXY'] || nil
    else
        config.proxy.enabled  = false
    end
  end
end
```

I have faster storage available elsewhere other than the default storage pool and I created a storage pool there called `vms`. Since I don't want to edit each Vagrantfile that someone sends me, I use this to ensure the storage pool is overridden for me without having to think about it. Secondly I used to work from behind a proxy as well as sometimes need to validate that some things didn't need external access so I needed to be able to control whether the proxy env was passed through. Both of these get merged into every project Vagrantfile that I run subsequently.

So that is the Vagrantfile I was referring to previously about not being picked up automatically where as when using the entrypoint it is possible to mount in the entire ~/.vagrant.d directory and the entrypoint would remap the directories.

However now that I think of it, it doesn't remap the Vagrantfile if it exists, so it doesn't currently make difference. Another small thing for me to look at if I can tidy up the entrypoint so that it is useful to handle this sort of thing for podman based execution as well.

Reply:
Thanks for for clarification. My use case is not so complex that I need to combine multiple Vagrantfiles but I can see how that can be useful in your circumstance.